docs/source/trainer/evaluation.rst

```diff
@@ -0,0 +1,63 @@
+Evaluation
+==========
+
+To track training progress, validation datasets can be provided to the
+Composer Trainer through the ``eval_dataloader`` parameter. The trainer
+will compute evaluation metrics on the evaluation dataset at a frequency
+specified by the the :class:`.Trainer` parameters ``validate_every_n_batches``
+and ``validate_every_n_epochs`` .
+
+.. code:: python
+
+   from composer import Trainer
+
+   trainer = Trainer(
+           ...,
+           eval_dataloader=my_eval_dataloader,
+           validate_every_n_batches = 100,  # Default is -1
+           validate_every_n_epochs = 1  # Default is 1
+   )
+
+The metrics should be provided by :meth:`.ComposerModel.metrics`.
+
+Multiple Datasets
+-----------------
+
+If there are multiple validation datasets that may have different metrics,
+use :class:`.Evaluator` to specific each pair of dataloader and metrics.
+This class is just a container for a few attributes:
+
+- ``label``: a user-specified name for the metric.
+- ``dataloader``: pytorch dataloader or our :class:`.DataSpec`.
+    See :doc:`Dataloaders</trainer/dataloaders>` for more details.
+- ``metrics``: :class:`torchmetrics.Metric` or :class:`torchmetrics.MetricCollection`.
```

Comment:
MetricCollection is not autolinking...

Reply:
yeah can't get it to link, after trying a few things. going to skip this for now.